src/main/java/com/minecolonies/achievements/AbstractAchievementPage.java

```diff
@@ -0,0 +1,18 @@
+package com.minecolonies.achievements;
+
+import net.minecraft.stats.Achievement;
+import net.minecraftforge.common.AchievementPage;
+
+/**
+ * Abstract achievement page
+ *
+ * @author Isfirs
+ * @since 0.2
+ */
+public abstract class AbstractAchievementPage extends AchievementPage
+{
+    public AbstractAchievementPage(final String name, final Achievement... achievements)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)